### ISSUE TITLE
Version 0.61.3 : Unknown exception for DynamoDB table for us-west-2 region

### ISSUE THREAD

**Issue by eburdon** [aws-cloudformation/cfn-lint#2314]
***cfn-lint version: (`cfn-lint --version`)***
cfn-lint 0.61.3

***Description of issue.***

Attempting to validate a template for the us-west-2 region fails when a DynamoDB table is defined as a resource.

Sample template:
```
{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Parameters": {},
  "Resources": {
    "ATableMapping": {
      "Type": "AWS::DynamoDB::Table",
      "Properties": {
        "BillingMode": "PAY_PER_REQUEST",
        "AttributeDefinitions": [
          {
            "AttributeName": "myAttributeName",
            "AttributeType": "S"
          }
        ],
        "KeySchema": [
          {
            "AttributeName": "myAttributeName",
            "KeyType": "HASH"
          }
        ]
      }
    }
  },
  "Outputs": {}
}

```

Running `cfn-lint mve.json -r us-west-2`  causes the following error:

```
E0002 Unknown exception while processing rule E3002: 'AWS::DynamoDB::Table.Json'
mve.json:1:1
```

This can also be reproduced by running `cfn-lint mve.json` when regional config Metadata exists the template, such as
```
 "AWSTemplateFormatVersion": "2010-09-09",
  "Metadata": {
    "cfn-lint": {
      "config": {
        "regions": ["us-west-2"]
      }
    }
  },
  "Parameters": {
```

The error **cannot** be reproduced by using different regions, such as `cfn-lint mve.json -r us-west-1` or `cfn-lint mve.json -r us-east-2`

Downgrading to `cfn-lint 0.61.2` appears to work as expected. Sample:
```
pip uninstall cfn-lint
pip install cfn-lint==0.61.2
cfn-lint mve.json -r us-west-2
```

I have checked the documentation referred to in the table for Supported Regions: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html 

During initial debugging, I also reviewed rule definitions here:
https://github.com/aws-cloudformation/cfn-lint/blob/main/docs/rules.md

**Comment by bspnmac** [comment]
@kddejong - can you advise?  We're having the same issue and it appears to be introduced by [this PR](https://github.com/aws-cloudformation/cfn-lint/pull/2306/files#diff-65deddaa4279993a3de6ef4eb9bfbf50335573839fd5b85bf91561894020a35bL99155)

the Cloudspecs for KeySchema are inconsistent across the us regions
![image](https://user-images.githubusercontent.com/76788566/181770153-58339e89-a006-4e6c-b20f-61998ee48bba.png)



is `"Type": "Json"` valid for `cfn-dynamodb-table-keyschema` ?

**Comment by kddejong** [comment]
Looking into this now.  I may patch it if I don't get a quick answer for resolution.

**Comment by kddejong** [comment]
Looks like this got fixed in #2308.  Going to get this released shortly but a `cfn-lint --update-specs` should fix it now

**Comment by kddejong** [comment]
Should be fixed in https://github.com/aws-cloudformation/cfn-lint/releases/tag/v0.61.4
